src/lib/Syntax.hs

```diff
@@ -1267,6 +1337,32 @@ pattern BinaryFunVal b1 b2 eff body =
 pattern IDo :: BinderP IType
 pattern IDo = Ignore IVoidType
 
+pattern NoLabeledItems :: LabeledItems a
+pattern NoLabeledItems <- ((\(LabeledItems items) -> M.null items) -> True)
+  where NoLabeledItems = LabeledItems M.empty
```

Comment:
Is this a branching pattern-only synonym? I couldn't get those to work some time ago 🤔

Reply:
Not sure what you mean? It's an explicitly-bidirectional pattern synonym using view patterns, if that's what you are asking. It matches only when the items are empty.